services/alert_params.go

```diff
@@ -0,0 +1,24 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package services
+
+// AlertParams defines alertmanager alert parameters.
+type AlertParams struct {
```

Comment:
So this is kinda model, but not stored in PostgreSQL. I'm not sure if we want to move it into models or keep there.
Or we can take a third-way and remove the whole struct, passing 3 parameters to methods.
@BupycHuk thoughts?

Reply:
I don't think that it should be moved somewhere else. We have similar structure like server.Params and scrapeConfigParams, so I think we can have extra one structure for params

Reply:
> We have similar structure like server.Params and scrapeConfigParams

Both are used only by a single service. That's unlike that `AlertParams`.

Reply:
> Or we can take a third-way and remove the whole struct, passing 3 parameters to methods.

@artemgavrilov let's do that ^. We can use unexported alertmanager.alertParams if needed, but should not be exported.

Reply:
Done